test/functional/apps/console/_autocomplete.ts

```diff
@@ -46,6 +46,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await PageObjects.console.promptAutocomplete();
         await PageObjects.console.pressEnter();
 
+        await retry.waitForWithTimeout('text area to contain comma', 25000, async () => {
+          const textAreaString = await PageObjects.console.getAllVisibleText();
+          return textAreaString.includes(',');
+        });
```

Comment:
I wonder if there is a way to check text area rendering is finished? This retry will wait for comma to appear in text, but I suspect the failure is due to some wrong text in place:
`Error: expected '}' to sort of equal ','`  

Were you able to reproduce it in Firefox?

Reply:
> @cuff-links did you run the fix with flaky-test-runner?

I did over the weekend. I did 50 executions and they all passed.

Reply:
@dmlemeshko What are your thoughts?